test/sketcherBasicSMILES.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * A very, very basic SMILES-like parser. No aromaticity,
+ * or zero-order bonds. No chirality/stereochemistry
+ *
+ * Do not use this as a basis for real SMILES parsers. It just lets
+ * us sidestep using a full chemistry toolkit when writing tests.
+ */
+
+
+#include <algorithm>
+#include <string>
+#include <stack>
+#include <vector>
+#include <unordered_map>
+#include <iostream>
+#include <algorithm>
+
+#include "../sketcherMinimizerMolecule.h"
+
+
+namespace schrodinger
+{
+
+sketcherMinimizerMolecule* approxSmilesParse(const std::string& smiles)
+{
+    const std::unordered_map<char, int> elements {
+        {'H', 1},
+        {'C', 6},
+        {'N', 7},
+        {'S', 16},
+        {'O', 8}
```

Comment:
I think these are the elements we'll be playing with most... maybe I'd add P since phosphates can render with a different layout (with 90 degrees angles), but we can do it when we need it

Reply:
yeah, I was thinking we could add when needed.